src/main/kotlin/ui/instance/creation/InstanceCreationDialog.kt

```diff
@@ -198,9 +207,20 @@ private fun CheckboxRow(
     value: Boolean,
     onValueChange: (Boolean) -> Unit,
 ) {
-    TooltipWrapper(tooltip = { TooltipText(tooltip) }) {
+    var state by remember { mutableStateOf(value) }
```

Comment:
CheckboxRow shouldn't keep internal state

Reply:
There is no way to do it differently though, as CheckBoxRow essentially becomes the checkbox